Caddyfile

```diff
@@ -1,5 +1,6 @@
 0.0.0.0
 proxy / registration-provider:3000
+proxy /team teams-provider:3000
```

Comment:
Not blocking but could probably be `/teams` pluraly because it's the `teams provider`